### ISSUE TITLE
Add a quick world hop option.

### ISSUE THREAD

**Issue by sinkag** [runelite/runelite#2731]
could have it in the side bar.

**Comment by sinkag** [comment]
I didnt mean it like hopping to a friends world. I mean implementing the "world switch" menu into the side bar. Or is that the same concept?

**Comment by Hydrox6** [comment]
Same issue; can't do it without faking inputs

**Comment by deathbeam** [comment]
Duplicate of #1617

**Comment by deathbeam** [comment]
Marked this as duplicate of 1617 because it is blocked on same thing and if we ever resolve it we probably also implemented other ways of quick hopping
